.github/dependabot.yml

```diff
@@ -12,3 +12,9 @@ updates:
     interval: weekly
     time: "10:00"
   open-pull-requests-limit: 10
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
This says weekly, but the PR description says monthly!

Reply:
Fixed.